InscryptionAPI/AscensionScreens/AscensionScreenManager.cs

```diff
@@ -0,0 +1,161 @@
+using DiskCardGame;
+using HarmonyLib;
+using UnityEngine;
+using System.Collections;
+
+namespace InscryptionAPI.AscensionScreens
+{
+    [HarmonyPatch]
+    public static class AscensionScreenManager
+    {
+        internal static List<Type> registeredScreens = new List<Type>();
+
+        internal static Dictionary<AscensionMenuScreens.Screen, AscensionRunSetupScreenBase> screens;
+
+        internal static AscensionMenuScreens.Screen initialScreen = AscensionMenuScreens.Screen.JournalSummary;
+
+        internal const int CUSTOM_SCREEN_START = 100;
+
+        private static string challengeScreenHoverText = "START RUN";
+
+        public static void AdvanceScreenQueue(bool forward=true)
```

Comment:
Is this meant to do something?

Reply:
This is vestigial from an old architecture. I missed it on my last pass through; it's been deleted.